Dockerfile

```diff
@@ -10,8 +11,8 @@ COPY data/ /pharmacy-data/
 
 RUN mongod --fork --logpath /var/log/mongodb.log --dbpath /data/db2 --smallfiles \
 
-    && mongoimport --jsonArray -d services -c services --file /pharmacy-data/pharmacy-data-large.json \
-    && mongo services --eval "db.services.createIndex({location:'2dsphere'})" \
+    && mongoimport --jsonArray -d services -c pharmacies --file /pharmacy-data/pharmacy-data.json \
+    && mongo services --eval "db.pharmacies.createIndex({location:'2dsphere'})" \
     && mongod --dbpath /data/db2 --shutdown \
```

Comment:
I think *all* calls to mongod should use --config to use the config created and not use the --dbpath and --logpath parameters. This means the logpath will need adding to the config.